rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -2829,8 +2829,8 @@ bool ppu_interpreter::BC(ppu_thread& ppu, ppu_opcode_t op)
 
 	ppu.ctr -= (bo2 ^ true);
 
-	const bool ctr_ok = bo2 | ((ppu.ctr != 0) ^ bo3);
-	const bool cond_ok = bo0 | (ppu.cr[op.bi] ^ (bo1 ^ true));
+	const bool ctr_ok = bo2 || ((ppu.ctr != 0) ^ bo3);
+	const bool cond_ok = bo0 || (ppu.cr[op.bi] ^ (bo1 ^ true));
```

Comment:
Nothing too wild in the assembly. The usual short-circuiting vs. added conditionals trade off. Probably not worth the time to micro-benchmark this, so I'll revert the change.